.storybook/preview.tsx

```diff
@@ -16,13 +16,27 @@ const preview: Preview = {
 export default preview;
 
 import theme from '../src/theme';
+import { useState } from 'react';
+import { LocalizationProvider } from '../src/providers';
 
 export const decorators = [
-  (Story) => (
-    <ThemeProvider theme={theme}>
-      <StyledEngineProvider injectFirst>
-        <Story />
-      </StyledEngineProvider>
-    </ThemeProvider>
-  ),
+  (Story) => {
+    const [selectedLocale, setSelectedLocale] = useState('en');
+    const [activeLocale, setActiveLocale] = useState<string | null>(null);
+
+    return (
+      <ThemeProvider theme={theme}>
+        <LocalizationProvider
```

Comment:
Cool!

Reply:
do you need to `yarn build && yarn run storybook` as the vercel command?

Reply:
Maybe.. I'll give that a try, thanks

Reply:
adding `yarn generate-strings` did the trick :)